user_guide.md

```diff
@@ -182,6 +182,23 @@ inception   LoadBalancer   10.35.255.136   ww.xx.yy.zz   9000:30936/TCP   28m
 
 In this example, you should be able to use the inception_client to hit ww.xx.yy.zz:9000
 
+### Serve a model using Seldon
+
+Install the seldon package 
+
+```
+ks pkg install kubeflow/seldon
+```
+
+Generate the core components
+
+```
+ks generate seldon seldon
+```
+
+Seldon allows complex runtime graphs for model inference to be deployed and covers models written using a wide range of tools. For an example end-to-end integration see the [kubeflow-seldon example](https://github.com/kubeflow/example-seldon).
```

Comment:
How about adding a link to Seldon docs. e.g. "For more information please refer to Seldon docs."